package.json

```diff
@@ -60,31 +60,29 @@
   ],
   "dependencies": {
     "@typescript-eslint/parser": "^4.0.0",
-    "eslint-config-standard": "^14.1.1"
+    "eslint-config-standard": "^16.0.0"
   },
   "peerDependencies": {
-    "eslint": ">=6.2.2",
-    "eslint-plugin-import": ">=2.18.0",
-    "eslint-plugin-node": ">=9.1.0",
-    "eslint-plugin-promise": ">=4.2.1",
-    "eslint-plugin-standard": ">=4.0.0",
-    "typescript": ">=3.9",
-    "@typescript-eslint/eslint-plugin": ">=4.0.1"
+    "eslint": "^7.12.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "typescript": "^3.9",
+    "@typescript-eslint/eslint-plugin": "^4.12.0"
```

Comment:
Yes, see change in standard:
"peer deps should use ^ operator" standard/eslint-config-standard@ffc3c75

Reply:
I've gone ahead and switched to option 2:
- peer deps use `>=` again
- test `Own peerDependencies include those of eslint-config-standard` now strips `^` from standard deps & strips `>=` from `ourRange`, then compares the version number _(note: it does also catch if either symbol is missing or different from expected since the array index will fail)_

This also meant that I could revert my changes to the other tests. The diff is now smaller & more focused.

(this update is all together in 1 commit so if you choose option 1 I can just revert that commit => I'm ready for either option!)